packages/neon/neon_talk/lib/src/pages/room.dart

```diff
@@ -123,4 +129,16 @@ class _TalkRoomPageState extends State<TalkRoomPage> {
       },
     );
   }
+
+  Widget buildUserStatus(spreed.Room room) {
+    final statusMessage = room.statusMessage;
+    if (statusMessage != null) {
+      return Text(
+        statusMessage,
+        style: Theme.of(context).textTheme.labelSmall,
+      );
+    }
+
+    return const SizedBox();
```

Comment:
Not a fan of returning the sized box

Reply:
I guess the whole thing could be inlined into the column or the whole column moved into a separate function :woman_shrugging:

Reply:
I'm open to both but please change this.